charts/jenkins-x-crds/templates/pipelineactivities-crd.yaml

```diff
@@ -289,15 +289,15 @@ spec:
                                 type: string
                               description:
                                 type: string
+                              message:
+                                type: string
                               name:
                                 type: string
                               startedTimestamp:
                                 format: date-time
                                 nullable: true
                                 type: string
                               status:
-                                description: ActivityStatusType is the status of an
```

Comment:
why is this getting removed?

Reply:
Because of this
```go
// ActivityStatusType is the status of an activity; usually succeeded or failed/error on completion
type (
	ActivityStatusType  string
	ActivityMessageType string
)
```
It should be like this
```go

type (
        // ActivityStatusType is the status of an activity; usually succeeded or failed/error on completion
	ActivityStatusType  string
	// ActivityMessageType is the message of an activity; usually succeeded or failed/error on completion
        ActivityMessageType string
)
```